dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/events/TrackedEntityInstanceServiceTest.java

```diff
@@ -244,6 +247,17 @@ public void testUpdateTeiByCompletingExistingEnrollmentAndAddNewEventsToSameEnro
 
     }
 
+    @Test
+    public void testSyncTeiFutureDatesForEnrollmentAndIncident() {
```

Comment:
New line for brackets ;)

Reply:
This classes entered by mistake :)
I am following the dhis2 code style in the others PR